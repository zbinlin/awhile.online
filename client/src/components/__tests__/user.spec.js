"use strict";

/* eslint-env jest */
import { h } from "preact";
import renderer from "react-test-renderer";
import User from "../user.jsx";

it("User renders correctly", () => {
    const tree = renderer.create(
        <User userInfo={{}} dispatch={() => {}} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
