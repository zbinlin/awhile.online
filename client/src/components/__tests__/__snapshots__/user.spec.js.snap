exports[`test User renders correctly 1`] = `null`;
